atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/HikariCPConnectionManager.java

```diff
@@ -304,7 +305,9 @@ private HikariDataSource getDataSourcePool() {
             }
         } catch (PoolInitializationException e) {
             log.error("[{}] Failed to initialize hikari data source: {}",
-                    connConfig.getConnectionPoolName(), connConfig.getUrl(), e);
+                    SafeArg.of("connectionPoolName", connConfig.getConnectionPoolName()),
```

Comment:
Check if this has hostnames, cause I don't think these are allowed?

Reply:
Yeah, so spec says: "Hostnames of Skylab-managed services (as they are already included in the Apollo envelope)" are safe. But I'm not convinced that e.g. if this is running with Oracle (would this be running on Oracle?) that we would be managing the hosts.

Reply:
In the previous PR to add some safe logging, this was added as Safe in other places, so I was following suit. 

`getConnectionPoolName` is constructed as `getConnectionPoolIdentifier()-getConnId()`, where the former is `db-pool` and the latter is `atlas` as defaults. Obviously these can be overridden in the config, but I'd assume those two fields, given that they are just config (and not passwords or anything of that calibre either) would be fine, but maybe not.

Reply:
ok